R/cnd-abort.R

```diff
@@ -665,17 +665,29 @@ utils::globalVariables(".internal")
 
 footer_internal <- function(env) {
   top <- topenv(env)
+  url_line <- NULL
+
   if (is_namespace(top)) {
-    pkg <- sprintf(" in the %s package", ns_env_name(top))
+    pkg <- ns_env_name(top)
+    url <- pkg_url_bug(pkg)
+    pkg_line <- sprintf(
+      "This is an internal error that was detected in the %s package.",
```

Comment:
I find it somewhat funny that we have all these nice cli-like helpers for formatting but still use `sprintf()` for interpolation